Portfolio Project - Desktop Version

BenjaminSemah opened the pull request:
**Desktop Version**
I have worked on the following;
- Created a desktop version of the portfolio project.
- Used media queries for the screen size of 1080 pixels.

BenjaminSemah committed:
Add style to navigation bar for desktop version.

BenjaminSemah committed:
Add style to headline section for desktop version

BenjaminSemah committed:
Add style to knowledge section for desktop version

BenjaminSemah committed:
Update style for main section for the desktop version

BenjaminSemah committed:
Update style of Homepage for Desktop version

BenjaminSemah committed:
Update style for desktop version

BenjaminSemah committed:
Update style for Homepage in the desktop version

BenjaminSemah committed:
Update style for homepage for the desktop version.

alonsofalconi commented on Index.html:
```diff
@@ -11,318 +11,355 @@
+						placeholder="Please leave your message here"
+						required
+					></textarea>
+					<input class="projectButton btn" type="submit" value="See Project" />
```
- [ ] [OPTIONAL] I would suggest updating the value of the button to "Send" instead of "See Project" according to the Figma design. I will mark this as [OPTIONAL] because it's a minor change but I strongly suggest updating this value :smiley: :raised_hands: 

  Your design:
  ![image](https://user-images.githubusercontent.com/81522435/144443638-0a479887-1789-4df4-b82e-d3053688a257.png)

  Figma design:
  ![image](https://user-images.githubusercontent.com/81522435/144443679-5283f9e7-b79f-4278-9ee7-34f6a16a00b5.png)

alonsofalconi commented on Index.html:
```diff
@@ -11,318 +11,355 @@
+						</li>
+					</ul>
+				</div>
+				<span class="mail">emailhere@email.com</span>
```
- [ ] [OPTIONAL] I think it could be a good idea to add the `mailto` attribute so the users can send you an email when clicking the link. You can see a guide [here](https://css-tricks.com/snippets/html/mailto-links/) :wink:

alonsofalconi reviewed (approved):
Hi @BenjaminSemah,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

![](https://media0.giphy.com/media/jJQC2puVZpTMO4vUs0/giphy.gif)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

BenjaminSemah committed:
Change value of form submit buttion

I have changed the value of the button from 'See Project' to 'Send' based on recommendations by code reviewer.
